src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -53,16 +68,16 @@
 
           <ul class="ml-2">
             <li>Use a white background and a legible font with contrasting font colour</li>
-            <li>PDF file type (maximum {{MAX_FILE_SIZE}} MB file size)</li>
+            <li>PDF file type (maximum {{ MAX_FILE_SIZE }} MB file size)</li>
           </ul>
 
           <div class="d-flex mt-4">
             <span class="font-weight-bold">Upload File</span>
             <FileUploadPdf
               ref="fileUploadRef"
+              :file="file"
```

Comment:
The `.sync` modifier on `v-bind` is deprecated. ESLint was complaining about that. I tested this by trying to upload a PDF. Seems to be working 👍 

![addstaffnotationdialogue  sync is deprecated](https://github.com/bcgov/business-filings-ui/assets/122301442/6e487715-b075-4f0c-8036-b89b95eb0064)

Reply:
Could you change it as recommended in the error message instead?

Reply:
Thanks for the change. Make sure you test with values going into the sub-component and back out (via same prop).

Reply:
Tested. Looks OK. I've been getting the `invalid PDF` error non-stop though. I'm assuming it's my fault as I probably am trying to upload invalid PDFs.

Reply:
Now Github Actions is giving us a warning. Hopefully it will go away when we upgrade to Vue/Vuetify 3.

As this tests OK (but please try with a valid PDF ;) ), I'm OK to resolve this conversation.

Reply:
By the way, there's a difference between `:file="file"` and the sync or v-model syntax.

The first binds a value going "in" to the sub-component only.

The second and third update the variable HERE (in this component) when the sub-component emits an update event.

Reply:
This issue is handled in another comment below.

Reply:
@chenhongjing I think you'll have to do the same thing Karim did.